.eslintrc.js

```diff
@@ -18,5 +18,6 @@ module.exports = {
         'max-len': 0,
         'no-console': 0,
         "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+        "no-unused-vars": 1,
```

Comment:
My intention is not to be forever. But, I believe for now - when we try-out new things a lot, this will speed-up our work. The `unused vars` are still reported as `warning`, though, just not stopping the build.